recipes/evalhyd-python/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "0.1.0.0" %}
+
+package:
+  name: evalhyd-python
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/evalhyd-python/-/archive/v{{ version }}/evalhyd-python-v{{ version }}.tar.gz
+  sha256: 339fe203c6dea5a8d5005de3f2dbe45019d9603508c238ba01ca0d80375fe452
+
+build:
+  number: 0
+  script_env:
+    - CPPFLAGS=-D_LIBCPP_DISABLE_AVAILABILITY -std=c++17    # [osx]
+    - EVALHYD_PYTHON_VENDOR_XTL=FALSE
+    - EVALHYD_PYTHON_VENDOR_XTENSOR=FALSE
+    - EVALHYD_PYTHON_VENDOR_XTENSOR_PYTHON=FALSE
+    - EVALHYD_PYTHON_VENDOR_EVALHYD_CPP=FALSE
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.15
+    - make                                                  # [unix]
+    - ninja                                                 # [win]
+    - python
+    - pybind11
+    - setuptools
+    - pip
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - pybind11
+    - xtl >=0.7.5
+    - xtensor >=0.24.6
+    - xtensor-python >=0.26.1
+    - libevalhyd ==0.1.0
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - evalhyd
+  commands:
+    - python -c "import evalhyd; assert evalhyd.__version__ == '{{ version }}'"
+    - python -c "import evalhyd; print(evalhyd.__doc__)"
+    - python -c "import evalhyd; print(evalhyd.evald.__doc__)"
+    - python -c "import evalhyd; print(evalhyd.evalp.__doc__)"
```

Comment:
Probably not needed.

Reply:
Yes, now there are some more meaningful tests afterwards, these are not very useful anymore. I will drop them.